BootloaderCommonPkg/Include/Guid/LoaderPlatformInfoGuid.h

```diff
@@ -46,6 +46,12 @@ extern EFI_GUID gLoaderPlatformInfoGuid;
 //Secure debug (0 - disabled, 1 - enabled)
 #define        HWSTATE_SECURE_DEBUG              BIT3
 
+// Hw Platform debug mode (0 - disabled, 1 - enabled)
+// This flag would be enabled when platform is pre-production or platform is in debug mode
+// This flag would be disabled when platform is in production.
+#define        HWSTATE_PLATFORM_DEBUG_ENABLED    BIT4

```

Comment:
Any difference between BIT3 and BIT4 ?

Reply:
BIT3 (HWSTATE_SECURE_DEBUG) is intended for secure debug token. Currently its not used by any of the platforms.